src/output/icons.rs

```diff
@@ -320,9 +357,16 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
 /// Mapping from lowercase file extension to icons.  If an image, video, or audio extension is add
 /// also update the extension filetype map.
 const EXTENSION_ICONS: Map<&'static str, char> = phf_map! {
+    "123dx"          => Icons::CAD,              // 󰻫
+    "3dm"            => Icons::CAD,              // 󰻫
+    "3g2"            => Icons::VIDEO,            // 
+    "3gp"            => Icons::VIDEO,            // 
+    "3gp2"           => Icons::VIDEO,            // 
+    "3gpp"           => Icons::VIDEO,            // 
+    "3gpp2"          => Icons::VIDEO,            // 
     "7z"             => Icons::COMPRESSED,       // 
     "a"              => Icons::OS_LINUX,         // 
-    "acc"            => Icons::AUDIO,            // 
```

Comment:
I think this change is not related to nerdfonts 3.0 and should be moved to #565

Reply:
Actually a lot of the changes here are not related to NF but I added here to make #565 ready to merge asap, but still there is no feedback there =(

Reply:
Changes require external upgrades are much harder to get merged. Bug fixes like this should be moved out

Reply:
@cafkafk would you like this change to be moved to #565?